rsync_to_rdisc.py

```diff
@@ -215,31 +214,48 @@ def rsync_server_remote(hpc_server, client, to_be_transferred, mount_path, run_f
     rsync_succes = True
 
     for run in to_be_transferred:
-        with open(settings.log_path, 'a', newline='\n') as log_file:
-            log_file_writer = writer(log_file, delimiter='\t')
+        with open(settings.log_path, "a", newline="\n") as log_file:
+            log_file_writer = writer(log_file, delimiter="\t")
             log_file_writer.writerows([["#########"], [f"Date: {date}"], [f"Run_folder: {run}"]])
 
         transfer_settings = to_be_transferred[run]
-        # settings per folder data type, such as remote input dir and local output dir, etc.
+        # Settings per folder data type, such as remote input dir and local output dir, etc.
         missing = check_if_file_missing(transfer_settings["files_required"], f"{transfer_settings['input']}/{run}", client)
 
         if missing:
             print(transfer_settings, rsync_succes, missing, run, run_file)
             rsync_succes = action_if_file_missing(transfer_settings, rsync_succes, missing, run, run_file)
             continue  # don't transfer the run if a required file is missing.
-        os.system((
-            "rsync -rahuL --stats {user}@{server}:{input}/{run} {mount_path}/{output}/ "
-            " 1>> {log_path} 2>> {errorlog_path} 2> {temp_error_path}").format(
-                user=settings.user,
-                server=hpc_server,
-                input=transfer_settings["input"],
-                run=run,
-                mount_path=mount_path,
-                output=transfer_settings["output"],
-                log_path=settings.log_path,
-                errorlog_path=settings.errorlog_path,
-                temp_error_path=settings.temp_error_path
-        ))
+
+        # Get include and exclude patterns as list to easily access key-value pair in unit test.
+        if transfer_settings.get("include", None):
+            include_patterns = [f"--include '{pattern}'" for pattern in transfer_settings["include"]]
+        else:
+            include_patterns = ""
+        if transfer_settings.get("exclude", None):
+            exclude_patterns = [f"--exclude '{pattern}'" for pattern in transfer_settings["exclude"]]
+        else:
+            exclude_patterns = ""
```

Comment:
To match the variable type when empty/filled
```suggestion
            exclude_patterns = []
```